### PR TITLE
Location based landing pages

### PR DESCRIPTION
**Pull request by shanimal08**
### References Issues:
https://app.asana.com/0/1138868228851596/1139843346254324

### Description:

New landing pages for organizations/cities/venues. 
3 new routes and relevant meta.

Some examples for testing (if you're using develop):

/cities/new-york
/organizations/4000-tix-org-fc689
/venues/arcade-empire-91580

### Release Screenshots / Video:
https://cl.ly/20df11d09b19

### Environment Variables:
 * No change

### CODE REVIEW

**Comment by Timo614** [comment]
One thing I forgot to mention earlier in asana is that since we allow multiple tags to point back to a given event, venue, organization, etc. some logic was put in place to allow one to be considered the main slug which others are redirected back to. This was done to prevent duplicate content SEO issues: https://moz.com/learn/seo/duplicate-content

In those cases instead of the response having a type it returns a `redirect` parameter with a URL which then the frontend would need to do a 301 redirect to (as we can't trigger redirects via the API like that since it's the frontend consuming it):
https://github.com/big-neon/bn-api/blob/master/api/src/helpers/application.rs#L62

To have that case locally you'd have to add a new record in the DB:
`insert into slugs (main_table_id, slug_type, main_table, slug) values ('790ab6d6-91f5-4525-849b-0efb682617c0', 'Event', 'Events', 'redirect-me');`

Note: it's not currently something that should be too much of a concern since in the initial landing page slugs branch we're not creating multiple slugs per venue, organization, event it's just something for the future where if we ever do because we change the event name, etc. it allows us to then redirect to the new page. So probably safe to ignore that case for now but will be something we'll want to support if we allow multiple slugs for a given one of those types in the future.

File: src/components/pages/venuelanding/Meta.js [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+				{ property: "og:type", content: "website" },
+				{ property: "og:url", content: "https://www.bigneon.com" },
+				{
+					property: "og:description",
+					content:
+						"Big Neon - Mobile-first event ticketing. We’re relentlessly focused on serving the needs of independent live music promoters."
+				},
+				{
+					property: "og:image",
+					content: "https://www.bigneon.com/site/images/bigneon-screen-app.png"
```
Comment (Jasonvdb) [inline src/components/pages/venuelanding/Meta.js]: Don't hard code, rather use `settings().webUrl`

File: src/components/pages/venuelanding/Meta.js [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+//Reference: https://github.com/nfl/react-helmet
+
+const Meta = props => {
+	return (
+		<Helmet
+			title={`Concert Tickets ${props.venueName} ${props.cityName}- Big Neon`}
+			meta={[
+				{
+					property: "og:title",
+					content: "Big Neon - The new standard in event ticketing"
```
Comment (Jasonvdb) [inline src/components/pages/venuelanding/Meta.js]: Don't hard code, rather use `settings().webUrl`

File: src/components/pages/venuelanding/Meta.js [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+				},
+				{ property: "og:type", content: "website" },
+				{ property: "og:url", content: "https://www.bigneon.com" },
```
Comment (Jasonvdb) [inline src/components/pages/venuelanding/Meta.js]: Don't hard code, rather use `settings().webUrl`

File: src/components/pages/venuelanding/Meta.js [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+				},
+				{ name: "twitter:site", content: "bigneon.com" },
+				{ name: "twitter:creator", content: "bigneon" },
+				{
+					name: "twitter:title",
+					content: "Big Neon - The new standard in event ticketing"
+				},
+				{
+					name: "twitter:image",
+					content: "https://www.bigneon.com/site/images/bigneon-screen-app.png"
```
Comment (Jasonvdb) [inline src/components/pages/venuelanding/Meta.js]: Don't hard code, rather use `settings().webUrl`

File: src/components/pages/venuelanding/Meta.js [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+				},
+				{
+					name: "description",
+					content: `Concert Tickets ${props.venueName} ${
+						props.cityName
+					}- Find tickets to live events and concerts on Big Neon.`
+				}
+			]}
+			link={[
+				{ rel: "canonical", href: "https://www.bigneon.com" },
```
Comment (Jasonvdb) [inline src/components/pages/venuelanding/Meta.js]: Don't hard code, rather use `settings().webUrl`

File: src/components/pages/venuelanding/Meta.js [hunk 6]
```diff
@@ -0,0 +1,54 @@
...
+				{
+					rel: "image_src",
+					href: "https://www.bigneon.com/site/images/home-logo.png"
```
Comment (Jasonvdb) [inline src/components/pages/venuelanding/Meta.js]: Don't hard code, rather use `settings().webUrl`

**Pull request merged**